From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Mon, 22 Oct 2012 21:56:07 +0200
Subject: Don't fail early if virDomainOpenGraphics fails

virt_viewer_app_open_connection invokes virDomainOpenGraphics which only works
for local connections. Returning after this failure makes us never reach the
ssh part so this unbreaks connections tunneled over ssh.
---
 src/virt-viewer-app.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/virt-viewer-app.c b/src/virt-viewer-app.c
index def52c5..28dc8cb 100644
--- a/src/virt-viewer-app.c
+++ b/src/virt-viewer-app.c
@@ -820,8 +820,7 @@ virt_viewer_app_default_activate(VirtViewerApp *self)
     VirtViewerAppPrivate *priv = self->priv;
     int fd = -1;
 
-    if (!virt_viewer_app_open_connection(self, &fd))
-        return -1;
+    virt_viewer_app_open_connection(self, &fd);
 
     DEBUG_LOG("After open connection callback fd=%d", fd);
 
